django/db/models/query.py

```diff
@@ -15,11 +15,10 @@
     DJANGO_VERSION_PICKLE_KEY, IntegrityError, connections, router,
     transaction,
 )
-from django.db.models import DateField, DateTimeField, sql
+from django.db.models import AutoField, DateField, DateTimeField, sql
 from django.db.models.constants import LOOKUP_SEP
 from django.db.models.deletion import Collector
 from django.db.models.expressions import Case, Expression, F, Value, When
-from django.db.models.fields import AutoField
```

Comment:
This change is not related with the ticket. Please revert it, we can make this cleanup (and probably similar imports cleanups) in a separate PR.

Reply:
That's fair. Will revert and raise a separate PR.

Reply:
For reference: https://github.com/django/django/pull/11689